substrate/client/rpc-spec-v2/src/transaction/transaction_broadcast.rs

```diff
@@ -100,13 +119,31 @@ where
 	<Pool::Block as BlockT>::Hash: Unpin,
 	Client: HeaderBackend<Pool::Block> + BlockchainEvents<Pool::Block> + Send + Sync + 'static,
 {
-	fn broadcast(&self, bytes: Bytes) -> RpcResult<Option<String>> {
+	async fn broadcast(
+		&self,
+		connection_details: ConnectionDetails,
+		bytes: Bytes,
+	) -> RpcResult<Option<String>> {
```

Comment:
we could probably change the return type `Option<String>` because this error is never used by the impl

however,  https://github.com/paritytech/jsonrpsee/blob/master/core/src/server/mod.rs#L153-#L171 IntoResponse is not implemented for Option<String> so never mind.